scripts/monitor/cromwell.metadata.2BQ.py

```diff
@@ -8,20 +8,34 @@
 import subprocess
 import pandas as pd
 
-#cromwell-task-monitor-bq Variables
-cromwellBaseUrl = "https://cromwell-v47.dsde-methods.broadinstitute.org"
+######################################
+# cromwell-task-monitor-bq Variables #
+######################################
 gcpProject = "broad-dsde-methods"
-cromwellTaskMonitorBqDirectory = "/Users/bshifaw/work/Broadinstitute/cromwell-task-monitor-bq/metadata/submit/"
 
-#Obtain workflow id
-print("/n" + "## Obtaining Wokflow IDs ##")
 HOME = os.getenv('HOME')
 cromshellDatabase = HOME + '/.cromshell/all.workflow.database.tsv'
 pythonScriptPath = sys.path[0]
 workflowID = pythonScriptPath + '/workflowIds.txt'
-workflowIDTemp = pythonScriptPath + '/workflowIds_temp.txt'
+workflowIDTemp = pythonScriptPath + '/workflowIds_temp.txt' 
+cromwellTaskMonitorBqDirectory = pythonScriptPath + "/cromwell-task-monitor-bq/metadata/submit/"
+#Clones the cromwell-task-monitor-bq repo if not present
+if not path.exists(cromwellTaskMonitorBqDirectory):
+	print("/n" + "## Clone cromwell-task-monitor-bq##")
+	os.system('git clone https://github.com/broadinstitute/cromwell-task-monitor-bq.git')
+	copyfile(pythonScriptPath + "/cromwell_metadata_bq", cromwellTaskMonitorBqDirectory + "/cromwell_metadata_bq")
+#Get cromwell server info from cromshell files
+with open(os.getenv('HOME')+"/.cromshell/cromwell_server.config", "r") as f:
```

Comment:
I believe you can use `HOME` you get above instead of getting the environment variable again.